libvmaf/src/feature/funque_motion.c

```diff
@@ -0,0 +1,74 @@
+/**
+ *
+ *  Copyright 2016-2020 Netflix, Inc.
+ *
+ *     Licensed under the BSD+Patent License (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         https://opensource.org/licenses/BSDplusPatent
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+#include <limits.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+#include <math.h>
+#include "funque_filters.h"
+/**
+ * Note: img1_stride and img2_stride are in terms of (sizeof(double) bytes)
+ */
+funque_dtype funque_image_sad_c(const funque_dtype *img1, const funque_dtype *img2, int width, int height, int img1_stride, int img2_stride)
```

Comment:
For the sake of readability, calling this `mad` would be more appropriate, since it is the Mean Absolute Difference, not Sum.